<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealParameterView</class>
 <widget class="QFrame" name="RealParameterView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>265</width>
     <height>290</height>
    </rect>
   </property>
   <property name="title">
    <string>Real Parameter</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="numberText">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="descriptionText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>45</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>45</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="identifierText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>44</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Identifier</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isWriteableBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Is Writeable</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isCommandCheckBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Is Command</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="valueBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="minBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="maxBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Format</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="formatText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>minBox</sender>
   <signal>editingFinished()</signal>
   <receiver>RealParameterView</receiver>
   <slot>updateEntityMin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxBox</sender>
   <signal>editingFinished()</signal>
   <receiver>RealParameterView</receiver>
   <slot>updateEntityMax()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valueBox</sender>
   <signal>editingFinished()</signal>
   <receiver>RealParameterView</receiver>
   <slot>updateEntityValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>formatText</sender>
   <signal>editingFinished()</signal>
   <receiver>RealParameterView</receiver>
   <slot>updateEntityFormat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isCommandCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>RealParameterView</receiver>
   <slot>updateEntityCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isWriteableBox</sender>
   <signal>clicked()</signal>
   <receiver>RealParameterView</receiver>
   <slot>updateEntityWriteable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>descriptionText</sender>
   <signal>editingFinished()</signal>
   <receiver>RealParameterView</receiver>
   <slot>updateEntityDescription()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateEntityDescription()</slot>
  <slot>updateEntityWriteable()</slot>
  <slot>updateEntityCommand()</slot>
  <slot>updateEntityValue()</slot>
  <slot>updateEntityMax()</slot>
  <slot>updateEntityMin()</slot>
  <slot>updateEntityFormat()</slot>
 </slots>
</ui>
